### PR TITLE
simulate observations

### PR DESCRIPTION
**Pull request by guilhermebodin**

### CODE REVIEW

File: docs/src/examples.md [hunk 1]
```diff
@@ -50,8 +50,14 @@ expected_value = forecast_expected_value(forec)
 plot!(plt, dates, expected_value, label = "Forecast")
 ```
 
-StateSpaceModels.jl handles missing values automatically, the package consider that observations with `NaN` are missing
-values.
+StateSpaceModels.jl also enables generating scenarios for the forecasting horizon
```
Comment (raphaelsaavedra) [inline docs/src/examples.md:53]:
```suggestion
StateSpaceModels.jl also enables simulating multiple scenarios for the forecasting horizon based on the estimated distributions.
```

File: src/forecast.jl [hunk 2]
```diff
@@ -39,4 +40,22 @@ function forecast(model::SSM, steps_ahead::Int;
     end
 
     return Forecast{Fl}(expected_value, covariance)
+end
+
+"""
+TODO
```
Comment (raphaelsaavedra) [inline src/forecast.jl:46]: Missing docstring

File: src/systems.jl [hunk 3]
```diff
@@ -244,3 +246,35 @@ function to_multivariate_time_variant(system::LinearUnivariateTimeVariant{Fl}) w
...
+
+to_multivariate_time_variant(system::LinearMultivariateTimeVariant) = system
+
+# Functions for simulations
+function simulate(sys::LinearUnivariateTimeInvariant{Fl}, 
+                  initial_state::Vector{Fl}, 
+                  n::Int;
+                  return_simulated_states::Bool = false) where Fl
+
+    m, m = size(sys.T)
```
Comment (raphaelsaavedra) [inline src/systems.jl:258]: Typo here

File: src/systems.jl [hunk 4]
```diff
@@ -244,3 +246,35 @@ function to_multivariate_time_variant(system::LinearUnivariateTimeVariant{Fl}) w
...
+    alpha = Matrix{Fl}(undef, n + 1, m)
+    alpha[1, :] = initial_state
+
+    # Sampling errors
+    chol_H     = sqrt(sys.H)
+    chol_Q     = cholesky(sys.Q)
+    standard_ε = randn(n)
+    standard_η = randn(n, size(sys.Q, 1))
+
+    # Simulate scenries
```
Comment (raphaelsaavedra) [inline src/systems.jl:270]:
```suggestion
    # Simulate scenarios
```

File: src/systems.jl (hunk 3 above)
Comment (guilhermebodin) [reply src/systems.jl:258]: where?

Comment (guilhermebodin) [reply src/systems.jl:258]: It should be `m = size(sys.T, 1)`

Comment (raphaelsaavedra) [reply src/systems.jl:258]: For a moment I thought it was supposed to be `m, n`, but either way the current way it's written is very confusing. `m = size(sys.T, 1)` is better.

**Pull request merged**